src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/EnumsShouldHaveZeroValue.Fixer.cs

```diff
@@ -38,7 +37,11 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
                 SyntaxNode node = root.FindNode(context.Span);
 
                 ISymbol declaredSymbol = model.GetDeclaredSymbol(node, context.CancellationToken);
```

Comment:
```suggestion
                ISymbol? declaredSymbol = model.GetDeclaredSymbol(node, context.CancellationToken);
```

Reply:
Sure, lets also hope we can soon move to the compiler API with annotations that will automatically flag this :)